src/modules/node-restore.ts

```diff
@@ -1,7 +1,7 @@
+import shell from "shelljs";
```

Comment:
I'm gonna have to look into re-enabling this setting/extension for sorting these. I know we haven't really had a hard rule about it on OSS projects or more recent client projects, but I do like the auto-formatting and putting less brain space into where I'm putting an import in.